dbfit-java/oracle/src/main/java/dbfit/environment/OracleEnvironment.java

```diff
@@ -222,7 +223,7 @@ public OracleEnvironment(String driverClassName) {
                 new OracleTimestampNormaliser());
         TypeNormaliserFactory.setNormaliser(oracle.sql.DATE.class,
                 new OracleDateNormaliser());
-        TypeNormaliserFactory.setNormaliser(oracle.sql.CLOB.class,
+        TypeNormaliserFactory.setNormaliser(oracle.jdbc.OracleClob.class,
```

Comment:
I think this is the main change. Can we keep that on only without all the rest of reformatting?

Reply:
@bhuesemann, where do you see these compilation warnings? I don't see any using `./gradle compileJava`  and we are supposed to have "-Xlint:unchecked", "-Xlint:deprecation" compiler options set.

Reply:
I see the warnings with (gradlew is important here to use the version of the project and not the installed local version). I have also a local jdk8 installed and in the execution path.
`gradlew clean assemble`

Concerning reformatting: this is a topic I'd like to discuss in general. I think it is time to organize a webkonf and/or setup a discussion/chat for those matters. Is there a group on mattermost oder slack?

Reply:
`Z:\workspace\dbfit (master -> origin)                                                                                                                                          
λ gradlew clean assemble                                                                                                                                                       
Starting a Gradle Daemon, 1 incompatible and 1 stopped Daemons could not be reused, use --status for details                                                                   
                                                                                                                                                                               
> Task :dbfit-java:oracle:compileJava                                                                                                                                          
Z:\workspace\dbfit\dbfit-java\oracle\src\main\java\dbfit\environment\OracleEnvironment.java:225: warning: [deprecation] CLOB in oracle.sql has been deprecated                 
        TypeNormaliserFactory.setNormaliser(oracle.sql.CLOB.class,                                                                                                             
                                                      ^                                                                                                                        
1 warning                                                                                                                                                                      
                                                                                                                                                                               
Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.                                                                                    
Use '--warning-mode all' to show the individual deprecation warnings.                                                                                                          
See https://docs.gradle.org/6.6.1/userguide/command_line_interface.html#sec:command_line_warnings                                                                              
                                                                                                                                                                               
BUILD SUCCESSFUL in 35s                                                                                                                                                        
110 actionable tasks: 110 executed                                                                                                                                             
Z:\workspace\dbfit (master -> origin)                                                                                                                                          
λ                                                                                                                                                                              `